### PR TITLE
removing unnecesary double parent proc calls

### PR DESCRIPTION
**Pull request by LudwigVonChesterfield**
## About The Pull Request

some places had unnecessary proc calls which I have removed

## Why It's Good For The Game

it's not and nobody cares

### CODE REVIEW

**Comment by XDTM** [comment]
i wonder if this fixes #39793

File: code/game/objects/items/devices/reverse_bear_trap.dm [hunk 1]
```diff
@@ -76,9 +76,6 @@
 					"<span class='userdanger'>You force open the padlock!</span>", "<i>You hear a single, pronounced click!</i>")
 					REMOVE_TRAIT(src, TRAIT_NODROP, REVERSE_BEAR_TRAP_TRAIT)
 				struggling = FALSE
-			else
-				..()
-			return
```
Comment (spookydonut) [inline code/game/objects/items/devices/reverse_bear_trap.dm]: this return needs to stay

**Review by spookydonut — CHANGES_REQUESTED** [review]

**Comment by LudwigVonChesterfield** [comment]
@XDTM no it doesn't

File: code/game/objects/items/devices/reverse_bear_trap.dm (hunk 1 above)
Comment (LudwigVonChesterfield) [reply code/game/objects/items/devices/reverse_bear_trap.dm]: it (now) does!

**Comment by actioninja** [comment]
Good job on all of these

**Comment by spookydonut** [comment]
good start but the feature i just wrote for spacemandmm detects another 48

**Pull request merged**